packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.scss

```diff
@@ -36,11 +40,13 @@ cbp-usa-banner {
     margin-right: var(--cbp-space-3x);
   }
 
-  .cbp-usa-banner-expand:not(#fakeId)  {
+  cbp-button.cbp-usa-banner-expand[fill='ghost'][color='primary']:not(#fakeId)  { 
+    //addition of fill & color in the selector is to up specificity to override button's dark mode var remapping correctly
+    
     --cbp-button-height: 1rem;
     --cbp-button-min-height: 1rem;
 
-    --cbp-button-padding: var(--cbp-space-2x);
+    --cbp-button-padding: var(--cbp-space-1x);
 
     --cbp-button-color: var(--cbp-color-interactive-secondary-light);
```

Comment:
these don't look right. The control is supposed to be blue like a link. See comments in the Jira ticket for focus values.

Reply:
updated color mappings to for enabled to interactive-primary-light and focus to interactive-focus-light (note: this var was explicitly requested for this mapping and results in focus being light blue not purple as in mockups)